app/api/ibc/[type]/packets.ts

```diff
@@ -194,11 +198,18 @@ export async function getPackets(request: NextRequest, apiUrl: string) {
 
 
     const key = `0x${channel.portId.split(".")[2]}-${channel.channelId}-${sequence}`;
+    const recvBlock = await providerTo.getBlock(recvPacketLog.blockNumber)
+
+    if (recvBlock!.timestamp < packets[key].createTime) {
+      continue
+    }
+
```

Comment:
Bug fix for when a recv packet event has a timestamp earlier than send packet event